### ISSUE TITLE
[Adaptive Cards> FlightUpdate.json]: Focus order is not appearing in logical sequence on passengers and seats name.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8978]
### Target Platforms

UWP

### SDK Version

1.0.1.0

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Application: Adaptive Cards Universal
OS Build: 24H2 (OS Build 26252.5000)
App Version: 1.0.1.0
Screen Reader: Narrator

**Repro Steps:**
1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ Windows+ Enter key and press caps lock +space bar key to turn on the scan mode.
3. Navigate to the 'FlightUpdate.json' through tab key and activate it.
4. Now, navigate to 'Passengers' name through up/down arrow key.
5. Observe the issue.

**Actual Result:**
Focus order is not appearing in logical sequence on passengers and seats name text when user navigates in the scan mode of narrator through up/down arrow key.

**Expected Result:**
Focus order is not appearing in logical sequence on passengers and seats name text when user navigates in the scan mode of narrator through up/down arrow key.

It should be something like Sarah Hum > 14A > Jeremy GoldBerg > 14B > EVAN > 14 C

**User Impact:**
Screen reader user will get impacted as user will have difficulty in navigating or in making proper track of the content.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section.

### Screenshots

<img width="960" alt="In scan mode, focus order is not in logical order" src="https://github.com/user-attachments/assets/eda513e4-c993-409a-8e7b-7a57c4caf55f">

https://github.com/user-attachments/assets/85fe4d43-3e21-4636-a27e-2a245830a647

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Removing Universal tags as product is getting deprecated.
